### DISCUSSION TITLE
How to deploy Datenstrom Yellow to Heroku?

### DISCUSSION THREAD

**Discussion by thaupc** [datenstrom/community#251] [Ask a question] [answered]
I think this little tip might help some people.
To deploy your yellow app to heroku, just install yellow on local host, config all the settings and themes...
Create `composer.json` with content:
```
{
	"require": {
  		"ext-mbstring": "*"
	}
}
```
Run `$ composer update` to generate `composer.lock` file.
Then you ready to push your app to heroku
`$ git init`
`$ heroku git:remote -a yourapp`
`$ git add .`
`$ git commit -am "make it better"`
`$ git push heroku master`

Make changes on posts and pages, then push via git as usual

**Comment by annaesvensson** [comment, marked as answer]
Thank you. 👍 Marking as answered/solved.
